src/prometheus_alert_rules/metrics_alert_rules.yaml

```diff
@@ -11,3 +12,65 @@ groups:
           summary: MySQL Down (instance {{ $labels.instance }})
           description: "MySQL instance is down\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}"
 
+      # 2.1.2
+      # customized: 80% -> 90%
+      - alert: MySQLTooManyConnections(>80%)
```

Comment:
Should be 90% here?

Reply:
yes, corrected in 0992fa9d9d51e4cf036a269394cabb588399140d